projects/go-tester/src/app/components/test-page-1/test-page-1.component.ts

```diff
@@ -95,4 +92,14 @@ export class TestPage1Component implements OnInit {
   cancelClick(): void {
     this.toasterService.toastError({ message: 'Cancel clicked!' });
   }
+
+  logRow(parentItem: any, childItem: any): void {
+    this.toasterService.toastInfo({
+      header: 'Child Row Clicked!',
+      message: `
+        Child Name: ${childItem.name.first}<br/>
+        Parent Name: ${parentItem.name.first}
+      `
+    });
+  }
```

Comment:
This is the method called from within the child row, you can see that you have access to both the specific child and parent objects.